### PR TITLE
More translations

### PR DESCRIPTION
**Pull request by dternyak**

### CODE REVIEW

File: common/components/Header/components/TabsOptions.jsx [hunk 1]
```diff
@@ -62,9 +64,8 @@ export default class TabsOptions extends Component {
                             {
                                 tabs.map((object, i) => {
                                         return (
-                                            <li className='nav-item'
-                                                key={i}
-                                                onClick={this.tabClick(i)}>
+                                            <li className={`nav-item ${(window.location.pathname === object.link || window.location.pathname.substring(1) === object.link) ? 'active': ''}`}
```
Comment (dternyak) [inline common/components/Header/components/TabsOptions.jsx]: fixes nav option active class.

File: common/translations/markup.js [hunk 2]
```diff
@@ -4,19 +4,28 @@ import React from 'react';
 const BOLD_REGEXP = /(\*\*)(.*?)\1/;
 const LINK_REGEXP = /\[([^\[]+)\]\(([^\)]+)\)/;
 
+function decodeHTMLEntities(text) {
```
Comment (dternyak) [inline common/translations/markup.js]: this I'm not too sure about - are we handling all edge cases? @crptm

File: common/components/Footer/index.jsx [hunk 3]
```diff
@@ -57,15 +55,6 @@ export default class Footer extends Component {
                                         TREZOR</a>
                                     </li>
                                 </ul>
-                                <ul>
```
Comment (crptm) [inline common/components/Footer/index.jsx]: I think we need to leave it, i'll add that in separate pr

File: common/components/Header/components/TabsOptions.jsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import React, {Component} from 'react';
-import {Link} from 'react-router';
-import translate from 'translations';
+import React, {Component} from "react";
```
Comment (crptm) [inline common/components/Header/components/TabsOptions.jsx]: eslint wants single quotes :(

File: common/components/Header/components/TabsOptions.jsx [hunk 5]
```diff
@@ -80,9 +81,9 @@ export default class TabsOptions extends Component {
                         this.state.showRightArrow &&
                         <a aria-hidden='true'
                            className='nav-arrow-right'
-                           onClick={() => this.scrollRight(100)}
-                           ng-mouseover='scrollHoverIn(false,2);' ng-mouseleave='scrollHoverOut()'>&#187;</a
```
Comment (crptm) [inline common/components/Header/components/TabsOptions.jsx]: @tayvano do we want these hover effects?

File: common/translations/markup.js (hunk 2 above)
Comment (crptm) [inline common/translations/markup.js]: We just need to handle everything that is in translation files. We want to move to something else after release anyway.

File: common/components/Footer/index.jsx [hunk 6]
```diff
@@ -1,14 +1,12 @@
-import React, {Component} from 'react'
+import React, {Component} from "react";
```
Comment (NessDan) [inline common/components/Footer/index.jsx]: Aren't our ESlint rules set to single-quote only?

File: common/components/Header/components/TabsOptions.jsx (hunk 1 above)
Comment (dternyak) [inline common/components/Header/components/TabsOptions.jsx]: yeah, went back and forth on if I should do that myself.

File: common/components/Footer/index.jsx (hunk 6 above)
Comment (dternyak) [inline common/components/Footer/index.jsx]: @NessDan Thanks - annoying IDE problems.

File: common/components/Header/components/TabsOptions.jsx (hunk 5 above)
Comment (dternyak) [inline common/components/Header/components/TabsOptions.jsx]: Its not working right now anyway, we can circle back.

File: common/components/Footer/index.jsx (hunk 3 above)
Comment (dternyak) [inline common/components/Footer/index.jsx]: You want me to put this back?

File: common/components/Header/components/TabsOptions.jsx [hunk 7]
```diff
@@ -63,10 +61,12 @@ export default class TabsOptions extends Component {
                         <ul className='nav-inner'>
                             {
                                 tabs.map((object, i) => {
+                                        // if the window pathname is the same or similar to the tab objects name, set the active toggle
+                                        let activeOrNot = (window.location.pathname === object.name || window.location.pathname.substring(1) === object.name) ? 'active' : '';
```
Comment (NessDan) [inline common/components/Header/components/TabsOptions.jsx]: Ah! Sorry for being a stickler but this is a `const` situation :X

After that this is good to merge.

**Pull request merged**